### DISCUSSION TITLE
Welcome to obsidian-fountain-live Discussions!

### DISCUSSION THREAD

**Discussion by chuangcaleb** [chuangcaleb/obsidian-fountain-editor#2] [Announcements]
### DO:

- 🪲 Send Bug Reports to the Issues tab — everything else should be here in Discussions!
- 🪢 Check if your discussion subject already exists before posting a duplicate! 
- 🏷️ Use the appropriate Discussions category and Labels!
- 🎁 Share appropriate alternatives! Tools are made for humans. Help a brother out!

**Comment by strahlkraft** [comment]
Hello, I am a newbie in obsidian. Therefore I am not an expert... I tried to use the fountain plug-in. I was able to setup obsidian so that it uses the correct format for screenplay text in the editor mode (INT. / (blabla)..., centered Text and so on) - but if I switch from editor to the display mode all the formats are gone... 

editor mode
<img width="699" alt="Bildschirmfoto 2023-11-14 um 21 12 02" src="https://github.com/chuangcaleb/obsidian-fountain-editor/assets/150852343/d0f3e644-d332-47dc-a788-cc754a520a63">

display mode
<img width="699" alt="Bildschirmfoto 2023-11-14 um 21 12 52" src="https://github.com/chuangcaleb/obsidian-fountain-editor/assets/150852343/9a84ec67-dd7f-4f91-80e9-637339de128d">

**Reply by chuangcaleb** [reply]
Hello! Thanks for writing in. Yeah, so as described in the README file, this plugin currently will only apply the styles for Live Preview mode but not Reading mode.

This may be a feature included later, but for now, you can use darakah's Obsidian Fountain plugin, which provides Fountain styling, not for Live Editor bu for Reading mode. 

Please do ask any further inquiries you may have!

**Reply by strahlkraft** [reply]
Hello! Thanks for your reply - yes I've read this. But I'm even not able to copy the text from the preview mode to another text editor with all the formatting. If I (or someone else) want to work with / read / write a screenplay it has to be done in the preview mode - am I right?

**Reply by chuangcaleb** [reply]
Oh hahaha. Well you see, the plug-in's formatting is all visual and not intrinsic. Obsidian is a **plain text editor** that uses special characters for formatting, which differs from say MS Word which is a **rich text editor** that will retain bold/italic formatting when you copy and paste. 

Plain-text is the core philosophy of solutions like Obsidian and Fountain. Storing/sharing it without any formatting is a MAJOR major plus. One plus is that you or anyone else can work on the same file with any compatible plain-text/Fountain editor (Highland 2 and slugline come to mind). 

So copying-without-formatting is intended, especially if you're writing Fountain. What text editor are you using? Any further explicit details?

But yes for now to get the styling, you have to use Live Preview mode and not Reading mode (see the Obsidian official help pages). :D perhaps try studying some videos of others writing Fountain and their workflow?

**Comment by strahlkraft** [comment]
Yes - that is all o.k. But even a pandoc export to PDF results in an unformatted text.
So I can write it in obsidian and will be happy with it. But the actors like to read a formatted paper ;-)

**Reply by chuangcaleb** [reply]
Oh yeah. Currently formatted PDF export isn't currently natively integrated with the plugin. I've currently left it to the user to find export options, I'm quite enjoying VS Code's [Better Fountain - Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=piersdeseilligny.betterfountain) extension. There's a lot of other options out there. 

Formatted PDF exports can be the next item on the list. You have my contact :D
